api/apiConfig.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+//const GITHUB_CLIENT_ID = process.env.GITHUB_CLIENT_ID || 'f7b0b0b0b0b0b0b0b0b0';
+//const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL || 'http://localhost:8001';
+
+const GITHUB_CLIENT_ID = "f7b0b0b0b0b0b0b0b0b0";
+const API_BASE_URL = "http://192.168.1.19:7437/";
```

Comment:
Can you switch it back to using the env?

Reply:
Done also added a .env.example